### PR TITLE
[IMP] account_financial_risk: Only block positive customer invoices validation

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You should not limit invoices that reduces the risk. They are:

- Normal invoices with negative amount.
- Refund invoices with positive amount.

**Comment by carlosdauden** [comment]
Those invoices that you comment, in addition to inconsistent, can cause other more serious problems that don't show the risk exceeded warning :wink:

**Comment by pedrobaeza** [comment]
That invoices can exist, so we should cover them. Don't think only on Spanish regulation (that are also admitted in some cases).

**Comment by carlosdauden** [comment]
Changes done

File: account_financial_risk/models/account_invoice.py [hunk 1]
```diff
@@ -11,7 +11,8 @@ class AccountInvoice(models.Model):
     def action_invoice_open(self):
         if self.env.context.get('bypass_risk', False):
             return super(AccountInvoice, self).action_invoice_open()
-        for invoice in self:
+        for invoice in self.filtered(lambda x: x.type in (
+                'out_invoice', 'out_refund') and x.amount_total_signed > 0.0):
```
Comment (pedrobaeza) [inline account_financial_risk/models/account_invoice.py:15]: This should be `<=`

Comment (pedrobaeza) [reply account_financial_risk/models/account_invoice.py:15]: Forget it. I have mixed the condition.

**Review by pedrobaeza — APPROVED** [review]
Fast-tracking this.

**Pull request merged**